packages/admin-ui/ui/src/domain/products/edit/index.tsx

```diff
@@ -18,7 +18,11 @@ const Edit = () => {
 
   const { getWidgets } = useWidgets()
 
-  const { product, status, error } = useAdminProduct(id || "")
+  const { product, status, error } = useAdminProduct(id || "", {
+    expand:
+      // default relations
+      "variants,variants.prices,variants.options,profiles,images,options,tags,type,collection",
```

Comment:
Question : isn't it enough to rely on the default relation from the end point?

Reply:
Sure, we can do that. In case the default relations are changed this page can break, so we just have to be aware of that.